administrator-guides/restoring-an-admin/README.MD

```diff
@@ -0,0 +1,71 @@
+# Restoring an Admin User
+
+You might accidently lost access to an important admin user (or the only admin in the server) and need to restore it without using another admin user. This guide will help you in this process.
+
+_Note: To do this, you will need access to the database_
+
+## Finding the database
+
+### Docker based installations
+
+Open Mongo shell within Mongo container
+
+Change into docker-compose directory (where your `docker-compose.yml` is located):
+
+```bash
+cd /opt/docker/Rocket.Chat
+docker-compose run mongo bash
+```
+
+Make sure to replace `mongo` with your MongoDB container name in case you use a different one.
+
+On the Mongo shell:
+
+```sql
+use rocketchat
+```
+
+Make sure to replace `rocketchat` with your actual Mongo database. If you're not sure how it's called either check your `docker-compose.yml` file or run:
+
+```sql
+show dbs
+```
+
+### Ubuntu Snaps
+
+Connect to MongoDB:
+
+```bash
+sudo rocketchat-server.mongo
+```
+
+Select Rocket.Chat Database:
+
+```sql
+use parties
+```
+
+## Updating the admin password
+
+You can either use a one time access token (that will require the user to change his password):
+
+```javascript
+db.getCollection('users').update({username:"administrator"}, {$set: { "services":{"loginToken":{"token":"some-token-id-that-you-will-use-to-login-once"}}, "requirePasswordChange":true} })

```

Comment:
You need to make sure to tell them how to actually go and use this.

They need to goto something like: http://{their server}/login-token/some-token-id-that-you-will-use-to-login-once

That will log them in and then let them set the password via the requirePasswordChange dialog